Makefile

```diff
@@ -0,0 +1,53 @@
+CURRENT_DIR = $(shell pwd)
+DOCKER_NAME ?= ansible-cvp
+DOCKER_TAG ?= latest
+
+.PHONY: build validate docker-image clean code-linting-ansible build-test code-import pre-commit code-validate code-import-ansible
```

Comment:
My (personal) preference is to have a self-documenting makefile as in here https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html

this just needs a boilerplate help target and then additional comments describing each target. It's makes makefiles much easier to use.

Reply:
Interesting. Will look at it and update this `Makefile` accordingly